scripts/scripts.js

```diff
@@ -371,29 +371,11 @@ export function createOptimizedPicture(src, alt = '', eager = false, breakpoints
 
 /**
  * Normalizes all headings within a container element.
- * @param {Element} el The container element
- * @param {[string]]} allowedHeadings The list of allowed headings (h1 ... h6)
  */
-export function normalizeHeadings(el, allowedHeadings) {
-  const allowed = allowedHeadings.map((h) => h.toLowerCase());
-  el.querySelectorAll('h1, h2, h3, h4, h5, h6').forEach((tag) => {
-    const h = tag.tagName.toLowerCase();
-    if (allowed.indexOf(h) === -1) {
-      // current heading is not in the allowed list -> try first to "promote" the heading
-      let level = parseInt(h.charAt(1), 10) - 1;
-      while (allowed.indexOf(`h${level}`) === -1 && level > 0) {
-        level -= 1;
-      }
-      if (level === 0) {
-        // did not find a match -> try to "downgrade" the heading
-        while (allowed.indexOf(`h${level}`) === -1 && level < 7) {
-          level += 1;
-        }
-      }
-      if (level !== 7) {
-        tag.outerHTML = `<h${level} id="${tag.id}">${tag.textContent}</h${level}>`;
-      }
-    }
+export function normalizeHeadings() {
+  // wrap all headings in a <div> with the class "region"
+  document.querySelectorAll('h1, h2, h3, h4, h5, h6').forEach((tag) => {
+    tag.outerHTML = `<div class="region container">${tag.outerHTML}</div>`;
```

Comment:
You're trying to update codes from boilerplate. If you think that this is something we need to update all other helix project, we need to update https://github.com/adobe/helix-project-boilerplate/blob/main/scripts/scripts.js#L377-L395 as well.
If you don't intend to do that, please undo this and make changes only after 
<img width="629" alt="image" src="https://user-images.githubusercontent.com/55804872/163066334-e3fa11f7-fbc2-4210-80b9-ace93568cf3a.png">

Reply:
Good catch. I decided it would be best to revert this, I overlooked that other places are using it and it's still useful. Went ahead and just added an additional function.

Reply:
Will you move your new function to after the holy line of `Edit above at your own risk`?